packages/ui/src/elements/withMergedProps/index.tsx

```diff
@@ -28,7 +28,7 @@ export function withMergedProps<ToMergeIntoProps, CompleteReturnProps>({
 }): React.FC<CompleteReturnProps> {
   // A wrapper around the args.Component to inject the args.toMergeArgs as props, which are merged with the passed props
   const MergedPropsComponent: React.FC<CompleteReturnProps> = (passedProps) => {
-    const mergedProps = deepMerge(toMergeIntoProps, passedProps)
+    const mergedProps = deepMerge(passedProps, toMergeIntoProps)
```

Comment:
Why the need to swap the order here?

Reply:
When this line was `deepMerge(toMergeIntoProps, passedProps)` and you check the merged output, the `path` value was getting overwritten as an empty string. When you swap the order, the merged props look correct. I can take another look however if this is not the right approach!

Reply:
I suspect that there's a better fix for this. Where is `path` being used?

Reply:
`path` is being used in the custom component that we are also passing to `withMergedProps`, in the component we're using `path` to set the relationship field value. Might be easier to pair and show you this.